src/test/kotlin/net/atos/zac/app/zaken/ZakenRESTServiceTest.kt

```diff
@@ -15,6 +15,7 @@ import io.mockk.mockk
 import io.mockk.runs
 import io.mockk.slot
 import io.mockk.verify
+import jakarta.enterprise.inject.Instance
```

Comment:
imports have been moved into alphabetical order, probably 'cause `javax` had an exclusion on that. Would we want to make the same exclusion for `jakarta` packages?

Reply:
Yes you are right. However we currently don't have an automated way to check this (similar to Detekt) so I suggest to pick this up in a follow up story/task. PS: Kotlin's Detekt does already have a rule for exactly this..